packages/Popover/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import Popover from './Popover';
```

Comment:
I did this but,  `export default from './Popover'`

Reply:
Yeah but that doesn't work in Typescript since we don't have the babel plugin for it.

Reply:
Ooh